development/src/composables/dbService.js

```diff
@@ -0,0 +1,82 @@
+import { appFirestore, appAuth, app } from "./firebaseConfig/config";
+import { setDoc, doc, addDoc, updateDoc, getDoc } from "firebase/firestore";
+
+async function addSession(userSessionData) {
+  const { sessionName, peerId, language, userName, peerName } = userSessionData;
+  if (!sessionName || !peerId || !language || !userName || !peerName) {
+    throw new Error("Kindly provide all fields");
+  }
+
+  const user = appAuth.currentUser;
+  if (user == null) {
```

Comment:
It'll for some reason claim the user is undefined, even if it's signed in.

Reply:
Keep the code, but debug it when it doesn't feel like it's working during test

Reply:
Alright.